packages/wrangler/src/__tests__/worker-namespace.test.ts

```diff
@@ -0,0 +1,267 @@
+import { mockAccountId, mockApiToken } from "./helpers/mock-account-id";
+import { setMockResponse, unsetAllMocks } from "./helpers/mock-cfetch";
+import { mockConsoleMethods } from "./helpers/mock-console";
+import { runInTempDir } from "./helpers/run-in-tmp";
+import { runWrangler } from "./helpers/run-wrangler";
+
+describe("worker-namespace", () => {
+	runInTempDir();
+	const std = mockConsoleMethods();
+	mockAccountId();
+	mockApiToken();
+
+	afterEach(() => {
+		unsetAllMocks();
+	});
+
+	it("should should display a list of available subcommands, for worker-namespace with no subcommand", async () => {
+		await runWrangler("worker-namespace");
+
+		// wait a tick for the help menu to be printed
+		await new Promise((resolve) => setImmediate(resolve));
+
+		expect(std).toMatchInlineSnapshot(`
+		Object {
+		  "debug": "",
+		  "err": "",
+		  "out": "wrangler worker-namespace
+
+		📦 Interact with a worker namespace
+
+		Commands:
+		  wrangler worker-namespace list                          List all Worker namespaces
```

Comment:
how do y'all feel about worker-namespace? Any alternate suggestions? cc @aaronlisman @rita3ko

Reply:
for posterity's sake :)
<img width="694" alt="image" src="https://user-images.githubusercontent.com/18808/176724595-20daa999-7bc6-4c18-abbc-41ab478a46f8.png">

Reply:
Yeah, only reason I went with `/dispatch` was because `/workers/namespaces` was already reserved for something that got deprecated/removed, but was still unavailable for our service. Don't necessarily need to have it in Wrangler.